Readme.md

```diff
@@ -17,6 +17,26 @@ code .
 ```
 The *switch name* (shown in the first column of `opam switch`) is the path to the goblint installation.
 
+### Project prerequisites
+
+The project must have:
+1. GobPie configuration file in project root with name "`gobpie.json`"
+2. Goblint configuration file ([see examples](https://github.com/goblint/analyzer/tree/master/conf))
+
+#### Gobpie configuration file
+
+Example configuration file `gobpie.json`:
+```
+{
+    "compilationDatabaseDirPath" : "build", 
```

Comment:
Now that we have GobPie's own configuration file, then this doesn't need to just be a single path to a compilation database, but anything Goblint can take as analysis target. So it could be renamed to `files` and be a list of strings, e.g.
* `["."]` (current directory should have the database)
* `["./build"]` (build directory should have the database)
* `["./build/compile_commands.json"]` (direct path to the database, not its directory)
* `["./01-assert.c"]` (single file for analysis without database)
* `["./01-assert.c", "extra.c"]` (multiple files for analysis without database)